modules/cluster-logging-deploy-multitenant.adoc

```diff
@@ -3,28 +3,37 @@
 // * logging/cluster-logging-deploying.adoc
 
 [id="cluster-logging-deploy-multitenant_{context}"]
-= Installing OpenShift Logging into a multitenant network
+= Installing OpenShift Logging with network isolation enabled
 
-If you are deploying OpenShift Logging into a cluster that uses multitenant isolation mode, projects are isolated from other projects. As a result, network traffic is not allowed between pods or services in different projects.
+If your cluster network provider enforces network isolation, you must allow network traffic between the projects that contain the operators that OpenShift Logging deploys.
 
-Because the OpenShift Elasticsearch Operator and the Red Hat OpenShift Logging Operator are installed in different projects, you must explicitly allow access between the  `openshift-operators-redhat` and `openshift-logging` projects. How you allow this access depends on how you configured multitenant isolation mode.
+Network isolation blocks network traffic between pods or services that are in different projects. Deploying OpenShift Logging installs the OpenShift Elasticsearch Operator in the  `openshift-operators-redhat` project and the Red Hat OpenShift Logging Operator in the `openshift-logging` project. Therefore, you must allow traffic between these two projects.
 
-.Procedure
+{product-title} offers two supported choices, OpenShift SDN and OVN-Kubernetes, for the default Container Network Interface (CNI) network provider:
 
-To allow traffic between the OpenShift Elasticsearch Operator and the Red Hat OpenShift Logging Operator, perform one of the following:
+* The OpenShift SDN provider enforces network isolation when configured to use the _network policy_ or _multitenant_ modes. It does not enforce network isolation when configured to use the _subnet_ mode.
 
-* If you configured multitenant isolation mode with the OpenShift SDN CNI plug-in set to the *Multitenant* mode, use the following command to join the two projects:
-+
-For example:
+* The OVN-Kubernetes provider enforces network isolation in a way that is similar to the OpenShift SDN's _network policy_ mode.
+
+.Procedure
+
+. If you are using the OpenShift SDN provider in *multitenant* mode, join the two projects. For example:
```

Comment:
```suggestion
. If you are using OpenShift SDN in *multitenant* mode, join the two projects. For example:
```

Reply:
If _cluster network provider_ is too long, I'd opt for just _OpenShift SDN_ or _OVN-Kubernetes_ for subsequent usage; just shortening to _provider_ isn't quite descriptive enough, so omitting it seems better in that case.